spec/brainstem/api_docs/formatters/open_api_specification/version_2/presenter_formatter_spec.rb

```diff
@@ -146,6 +147,84 @@ module Version2
                 stub(presenter).conditionals { conditionals }
               end
 
+              context 'with associations present' do
+                context 'when association type is belongs_to or has_one' do
+                  before do
+                    presenter_class.associations do
+                      association :task, Task,
+                        type: :belongs_to
+
+                      association :user, User,
+                        foreign_key: :user_id,
```

Comment:
Lets rename `foreign_key` to something better. Maybe `association_field_key` or `model_field_key`...